pom.xml

```diff
@@ -57,7 +57,7 @@
         <spring.security.version>4.0.3.RELEASE</spring.security.version>
```

Comment:
I will update with spring security 4.0.4.RELEASE version and spring to
4.2.5.RELEASE. these are the latest in maven repository (released february 2016)

http://mvnrepository.com/artifact/org.springframework.security/spring-security-core
http://mvnrepository.com/artifact/org.springframework/spring-context